checkpoint.go

```diff
@@ -63,7 +63,7 @@ checkpointed.`,
 			fatalf("Container cannot be checkpointed in %s state", status.String())
 		}
 		options := criuOptions(context)
-		if !options.LeaveRunning || !options.PreDump {
+		if !options.LeaveRunning && !options.PreDump {
```

Comment:
or will it look better like this?
```go
if !(options.LeaveRunning || options.PreDump) {
```

Reply:
For me that is more readable. Not really important to change it, just if you like to push it once more.